docs/sources/release-notes/v2-4.md

```diff
@@ -47,16 +49,39 @@ Here is a list of limit defaults that have changed:
 
 We have enabled the [in-memory FIFO cache by default](https://github.com/grafana/loki/pull/4519) when no existing external cache is configured. This helps improve Loki's performance, however it will also allow Loki to consume more memory.
 
+### v2.4.2 changes to defaults
+
+Here is a list of limit defaults that have changed in v2.4.2:
```

Comment:
Shouldn't we mention the PR that made these changes? https://github.com/grafana/loki/pull/5077